x-pack/plugins/apm/common/agent_configuration/setting_definitions/general_settings.ts

```diff
@@ -86,26 +86,12 @@ export const generalSettings: RawSettingDefinition[] = [
       'xpack.apm.agentConfig.captureHeaders.description',
       {
         defaultMessage:
-          'If set to `true`, the agent will capture request and response headers, including cookies.\n\nNOTE: Setting this to `false` reduces network bandwidth, disk space and object allocations.'
+          'If set to `true`, the agent will capture HTTP request and response headers (including cookies), as well as message headers/properties when using messaging frameworks (like Kafka).\n\nNOTE: Setting this to `false` reduces network bandwidth, disk space and object allocations.'
       }
     ),
     excludeAgents: ['js-base', 'rum-js']
   },
 
-  // LOG_LEVEL
-  {
-    key: 'log_level',
-    type: 'text',
-    defaultValue: 'info',
-    label: i18n.translate('xpack.apm.agentConfig.logLevel.label', {
-      defaultMessage: 'Log level'
-    }),
-    description: i18n.translate('xpack.apm.agentConfig.logLevel.description', {
-      defaultMessage: 'Sets the logging level for the agent'
-    }),
-    excludeAgents: ['js-base', 'rum-js', 'python']
-  },
-
```

Comment:
@eyalkoren unless we decide to remove it from `dotnet` and `ruby`, we should keep it. @formgeist WDYT?

Reply:
I don't know what is the cutoff to use in agent-dedicated config description vs. in this `general_settings` one. Otherwise it means increasing this `excludeAgents` to contain 5 agents. 
It's really up to you, just a suggestion.

Reply:
I have changed it to `includeAgents` instead of `excludeAgents` to make it easy to read. 
```
includeAgents: ['dotnet', 'ruby']
```
But I'm not against removing it if any agent is going to use it. 😃

Reply:
@cauemarcondes so you have both options now? Include and exclude? Isn't there an option to only show for one agent, I believe Java had a bunch that is Java-specific?

Reply:
@formgeist no, I removed the `excludeAgents` and replaced it for `includeAgents`, so this option will be available for `dotnet` and `ruby` only. Java is the only one who has its own config file, it must be because of what you said (`Java had a bunch that is Java-specific `). 
My question here is if we can remove the log_level property from all agents., if yes we can keep what @eyalkoren did here.

Reply:
We should keep the `log_level` option for `ruby` and `dotnet` since they have this active in their dynamic config options.

Reply:
Done